Mention `testcontainers-bom` in "Maven dependencies" section of… (#2178)
testcontainers/testcontainers-java: docs/index.md

    <scope>test</scope>
</dependency>
```

> To avoid specifying the version of each dependency, you can use `BOM` or `Bill Of Materials`.

Reviewer: ```suggestion

You can also [check the latest version available on Maven Central](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.testcontainers%22).

### Managing versions for multiple Testcontainers dependencies

To avoid specifying the version of each dependency, you can use a `BOM` or `Bill Of Materials`.
```

> Using `Maven` you must add the following to `dependencyManagement` section in your `pom.xml`:

Reviewer: ```suggestion
Using Maven you can add the following to `dependencyManagement` section in your `pom.xml`:
```

```xml tab='Maven'
<dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>org.testcontainers</groupId>
            <artifactId>testcontainers-bom</artifactId>
            <version>{{latest_version}}</version>
            <type>pom</type>
            <scope>import</scope>
        </dependency>
    </dependencies>
</dependencyManagement>
```
and then use dependencies without specifying a version:

```xml tab='Maven'
<dependency>
    <groupId>org.testcontainers</groupId>
    <artifactId>mysql</artifactId>
    <scope>test</scope>
</dependency>
```

> Using `Gradle` you must add the following to `dependencies` section in your `build.gradle`:

Reviewer: ```suggestion
Using Gradle 5.0 or higher, you can add the following to the `dependencies` section in your `build.gradle`:
```

```groovy tab='Gradle'
implementation platform('org.testcontainers:testcontainers-bom:{{latest_version}}') //import bom
testImplementation('org.testcontainers:mysql') //no version specified
```

> You can also [check the latest version available on Maven Central](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.testcontainers%22).

Reviewer: ```suggestion
```
